src/schema/users.ts

```diff
@@ -1353,11 +1354,13 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         };
       }
 
-      const hasClearedStreak = await checkAndClearUserStreak(
+      const hasClearedStreak = await queryReadReplica(
         ctx.con,
-        info,
-        streak,
+        ({ queryRunner }) => {
+          return checkAndClearUserStreak(queryRunner.connection, info, streak);
```

Comment:
Tnx for the changes, from my testing it might be better to pass `queryRunner`, see https://github.com/dailydotdev/daily-api/pull/2340, but not blocking. Let's see tomorrow.

Reply:
@capJavert It makes only a slight difference.
(See the last before update now doesn't use a new `pg.connect` but due to needing both the RR and master it's a bit tricky to omit even more.

Let me know what you think.

| Before | After |
|--------|--------|
| ![Screenshot 2024-10-15 at 09 24 52](https://github.com/user-attachments/assets/8fb05252-34d5-40e8-b4e4-1510fec2fef2) | ![Screenshot 2024-10-15 at 09 37 41](https://github.com/user-attachments/assets/c486908e-ca4a-4b22-86b5-ecd93d6de23b) |

Reply:
yeah, not blocking here since its a one call either way